### ISSUE TITLE
Feature Request: Show numbers of games in Explorer Panel

### ISSUE THREAD

**Issue by bluddy** [JosefNemec/Playnite#1343]
I love the Explorer panel. As a bonus, it would be great if each option (e.g. genre) showed the number of matching games before you even clicked on it.

**Comment by dreampillet** [comment]
Agreed. This would make the explorer panel a lot more practical! I think it should work like a column heading, allowing you to sort the list based on the count.

I did a mockup, trying to style it like the group count on the list panel:

<img width="333" height="240" alt="Image" src="https://github.com/user-attachments/assets/2fbc5197-437c-457c-aa5d-0e9f74e8d3c0" />
